include/classes/DAO/UserGroupsDAO.class.php

```diff
@@ -35,13 +35,13 @@ class UserGroupsDAO extends DAO {
 	 */
 	public function Create($title, $description)
 	{
-		global $addslashes, $msg;
+		global $msg;
```

Comment:
If you search `$addslashes` across the code base, they appear at various code comments. Could you sync them up to `$this->addSlashes`? Thanks.